An error on `osx-64` and `linux-64`:
```
  spacy/matcher/levenshtein.c:2248:3: error: no member named 'use_tracing' in 'struct _PyCFrame'
    __Pyx_TraceCall("make_levenshtein_compare", __pyx_f[0], 31, 0, __PYX_ERR(0, 31, __pyx_L1_error));
```

--- a/abs.yaml
+++ b/abs.yaml
@@ -1,3 +1,3 @@
 channels:
-  - https://staging.continuum.io/prefect/fs/cython-blis-feedstock/pr6/ff3408c
-  - https://staging.continuum.io/prefect/fs/thinc-feedstock/pr17/10d277f
+  - https://staging.continuum.io/prefect/fs/cython-blis-feedstock/pr6/a90ed26
+  - https://staging.continuum.io/prefect/fs/thinc-feedstock/pr17/1c1b99a

--- a/recipe/meta.yaml
+++ b/recipe/meta.yaml
@@ -31,7 +31,7 @@ requirements:
     - numpy >=2.0.0,<2.1.0
     - setuptools
     - wheel
-    - cython >=0.25,<3.0
+    - cython 3
     - murmurhash >=0.28.0,<1.1.0
     - cymem >=2.0.2,<2.1.0
     - preshed >=3.0.2,<3.1.0
@@ -52,7 +52,7 @@ requirements:
     - weasel >=0.1.0,<0.5.0
     # Third-party dependencies
     - numpy >=2.0.0
-    - tqdm >=4.38.0,<5.0.0
+    - tqdm >=4.,<3.038.0,<5.0.0
     - requests >=2.13.0,<3.0.0
     # Exclude pydantic 1.8.2 due to issues with python 3.11
     - pydantic >=1.7.4,!=1.8,!=1.8.1,!=1.8.2,<3.0.0

--- a/recipe/meta.yaml
+++ b/recipe/meta.yaml
@@ -52,7 +52,7 @@ requirements:
     - weasel >=0.1.0,<0.5.0
     # Third-party dependencies
     - numpy >=2.0.0
-    - tqdm >=4.,<3.038.0,<5.0.0
+    - tqdm >=4.38.0,<5.0.0
     - requests >=2.13.0,<3.0.0
     # Exclude pydantic 1.8.2 due to issues with python 3.11
     - pydantic >=1.7.4,!=1.8,!=1.8.1,!=1.8.2,<3.0.0

